clippy_lints/src/methods/mod.rs

```diff
@@ -425,6 +430,11 @@ declare_clippy_lint! {
     /// # let vec = vec![1];
     /// vec.iter().find(|x| **x == 0).is_some();
     /// ```
+    /// Could be written as
+    /// ```rust
+    /// # let vec = vec![1];
+    /// vec.iter().any(|x| **x == 0);
```

Comment:
This fails on travis with

```
error[E0614]: type `{integer}` cannot be dereferenced
 --> doctest.rs:27:20
  |
4 | vec.iter().any(|x| **x == 0);
  |                    ^^^
```

You should be able to run it locally with `cargo test --doc SEARCH_IS_SOME`

Reply:
but then the lint shouldn't make this a recommendation :confused: 
>= note: replace `find(|x| **x == 0).is_some()` with `any(|x| **x == 0)`

Reply:
cc #4453